deployments/lens.json

```diff
@@ -0,0 +1,23 @@
+{
+  "DiamondCutFacet": "0x3bcF271A7b0AB47D83dCc257674E88F01592DC41",
+  "LiFiDiamond": "0xF3B20515d9B193531c48E47c18aF16d1e5d28f9a",
+  "DiamondLoupeFacet": "0xB81b36D7909C4bfC332219454b3f0331a5633533",
+  "OwnershipFacet": "0x6a8417a9f7B5eF19e1f0007BA3aFaF42B0Fc39d6",
+  "DexManagerFacet": "0x56b8A6E7a8f058f74C42d26936F335fC65fe3962",
+  "WithdrawFacet": "0x7a50b40dbc33BE22FBF941e4eDbC26C9838a5896",
+  "PeripheryRegistryFacet": "0x1895985Ff263B502B40f984b6976DB845658B767",
+  "GenericSwapFacet": "0xD69298878d2615DF57B4FADDd402f63a96A9bFd1",
+  "GenericSwapFacetV3": "0xA16895a5be2375152c02Db599a6bB1Dd881E3342",
+  "CalldataVerificationFacet": "0x25A13FeE9b16114C9F281792Eed3Dfee29938365",
+  "EmergencyPauseFacet": "0x89a8f3d7F2c776022aC77E6818DefA109f9e5f25",
+  "AccessManagerFacet": "0xa9347E8f6Fa500D27b0E3482525dA6b95bED1684",
+  "ERC20Proxy": "0x6C251afA2ca69b53BF4588B63f2820a91672A6D3",
+  "Executor": "0xAb30c390671F509D89527F2FfEa6F9a072E3B636",
+  "FeeCollector": "0x6B6A13e1a04D0a899e0c9b49AaB966a0C5A7f9f3",
+  "TokenWrapper": "0x13A0486DCeEB9908D09bAD8136c0512D529383AC",
+  "LiFiDEXAggregator": "0x642eeb39B287bB7809043aaE89bcfac2F409737D",
+  "AcrossFacetV3": "0x42B178f9768234463DeCa66AFe4b9e0be67279b2",
+  "ReceiverAcrossV3": "0xe2868D398a48FA3a03d8Df94442Dc602332Fe4D4",
+  "GasZipFacet": "0xd740aC105578934556b369E6239D1A3fe37C9593",
+  "GasZipPeriphery": "0x4471933c5aB5d913a4C6542CE2DFcB964678b5cc"
```

Comment:
I believe GasZipPeriphery address was not added to DEX whitelist (and thus also not whitelisted)